astropy/coordinates/earth.py

```diff
@@ -368,13 +368,33 @@ def to_geocentric(self):
     def itrs(self):
         """
         Generates an `~astropy.coordinates.ITRS` object with the coordinates of
-        this object.
+        this object at the default ``obstime``.
         """
-        #potential circular imports prevent this from being up top
+        #circular imports prevent this from being up top
         from .builtin_frames import ITRS
 
         return ITRS(x=self.x, y=self.y, z=self.z)
 
+    def get_itrs(self, obstime):
```

Comment:
If we define `obstime=None` here (or whatever is necessary to make it use the "default obstime"), you can just do `itrs = property(get_itrs)` and remove the duplicate definition above.